spec/helpers/container_summary_helper_spec.rb

```diff
@@ -1,6 +1,6 @@
 describe ContainerSummaryHelper do
-  REL_HASH_WITH_LINK = [:label, :image, :value, :link, :title]
-  REL_HASH_WITHOUT_LINK = [:label, :image, :value]
+  REL_HASH_WITH_LINK = [:label, :image, :value, :link, :title].freeze
+  REL_HASH_WITHOUT_LINK = [:label, :image, :value].freeze
```

Comment:
These constants should probably be moved to lets to not contaminate the global space (but yeah, maybe for a separate PR...it's up to you).